style.css

```diff
@@ -445,6 +443,145 @@ nav {
   display: none;
 }
 
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background: var(--logo);
+  opacity: 0.9;
+  backdrop-filter: blur(4px);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.overlay .wrapper {
+  position: relative;
+  width: 90%;
+  height: 90%;
+  background: var(--white);
+  border-radius: 8px;
+  display: grid;
+  grid-template-rows: 1fr 1fr 2fr 1fr 1fr;
+  box-sizing: border-box;
+}
+
+.wrapper .wrapper-head {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  padding: 1em 1em 0.5em;
+}
+
+.wrapper-head h2 {
+  padding: 15px 15px 0;
+  font-size: 32px;
+  font-weight: 700;
+  color: var(--h1);
+}
+
+.wrapper .wrapper-sub-head {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 10px;
+  padding-left: 2em;
+}
+
+.close-btn {
+  color: #67798e;
+  font-size: 2em;
+  text-align: right;
+  cursor: pointer;
+}
+
+.wrapper .wrapper-img {
+  display: grid;
+  grid-template-columns: 1fr;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.wrapper .wrapper-img .work-image {
+  width: 90%;
+  margin: 0 auto;
+  margin-top: 5%;
+}
+
+.wrapper .wrapper-text {
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: 1em;
+  padding-right: 1em;
+  display: grid;
+  grid-template-rows: 1fr;
+}
+
+.wrapper .wrapper-text .long-paragraph {
+  font-size: 12px;
+  margin-bottom: 10px;
+  color: var(--paragraph);
+}
+
+.wrapper .wrapper-text span {
+  margin-right: 10px;
+  padding: 5px;
+  background: var(--language);
+  border-radius: 4px;
+  color: var(--logo);
+  font-size: 12px;
+}
+
+.wrapper .wrapper-text hr {
+  border: none;
+  border-bottom: 1px solid #babbbd;
+  margin-top: 10px;
+}
+
+.wrapper .wrapper-btns {
+  display: flex;
+  justify-content: center;
+}
+
+.wrapper .wrapper-btns button {
+  margin: 0.5em 1em;
+  padding: 1em;
+  padding-right: 2.3em;
+  background: transparent;
+  text-decoration: none;
+  border: 1px solid var(--logo);
+  border-radius: 8px;
+  color: var(--logo);
+  font-size: 12px;
+  text-align: left;
+}
+
+.wrapper .wrapper-btns .btnA {
+  background-image: url(./images/live.png);
+  background-repeat: no-repeat;
+  background-position-x: 95%;
+  background-position-y: center;
+}
+
+.wrapper .wrapper-btns .btnB {
+  background-image: url(./images/source.png);
+  background-repeat: no-repeat;
+  background-position-x: 95%;
+  background-position-y: center;
+}
+
+.wrapper .wrapper-btns i {
+  font-size: 12px;
+}
+
+.mdl-fullscreen {
+  display: none;
+}
+

```

Comment:
- [ ] You did a great job fixing the position of your desktop popup but it still looks very off from the Figma template. Kindly try to follow the Figma design as much as possible. Your popup design has some issues like the background is a little bit transparent, the paddings of the desktop tech tags are no matching, the paddings of your mobile 'See Live'/'See Source' buttons are not matching, the image sizing in the mobile design is off, etc...
Your design:
<img width="423" alt="Screen Shot 2022-07-22 at 4 24 53 PM" src="https://user-images.githubusercontent.com/61919617/180515069-fe73fb8f-f4d6-4112-96e7-2e8a902f1ed0.png">
<img width="1412" alt="Screen Shot 2022-07-22 at 4 24 22 PM" src="https://user-images.githubusercontent.com/61919617/180515310-0ca54118-0073-44a6-95fc-b4be88d6eac7.png">